wsmaster/che-core-api-factory-github/src/main/java/org/eclipse/che/api/factory/server/github/GithubApiClient.java

```diff
@@ -132,6 +137,50 @@ public GithubPullRequest getPullRequest(
         });
   }
 
+  /**
+   * Returns the last commit of the branch.
+   *
+   * <p>GitHub REST API documentation: https://docs.github.com/en/rest/commits/commits
+   *
+   * @param user
+   * @param repository
+   * @param branch
+   * @param authenticationToken
+   * @return
+   * @throws ScmBadRequestException
+   * @throws ScmItemNotFoundException
+   * @throws ScmCommunicationException
+   * @throws URISyntaxException
```

Comment:
Please add description to the empty references or remove them.

Reply:
Done